src/imports.rs

```diff
@@ -41,19 +42,13 @@ impl<'a> FmtVisitor<'a> {
         // 2 = } + ;
         let used_width = indent + 2;
 
-        let remaining_budget = if used_width >= budget {
-            return None;
-        } else {
-            budget - used_width
-        };
-
```

Comment:
You still need some checks on the width vs budget - if you don't you could get underflow errors.